tunnel_routes.go

```diff
@@ -0,0 +1,137 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// TunnelRoute is the full record for a route.
+type TunnelRoute struct {
+	Network    string    `json:"network"`
+	TunnelId   string    `json:"tunnel_id"`
+	TunnelName string    `json:"tunnel_name"`
+	Comment    string    `json:"comment"`
+	CreatedAt  time.Time `json:"created_at"`
+	DeletedAt  time.Time `json:"deleted_at"`
```

Comment:
these `time.Time` references should be pointers

```suggestion
	CreatedAt  *time.Time `json:"created_at"`
	DeletedAt  *time.Time `json:"deleted_at"`
```